pkg/uploader/kopia/snapshot.go

```diff
@@ -245,7 +245,7 @@ func reportSnapshotStatus(manifest *snapshot.Manifest, policyTree *policy.Tree)
 	var errs []string
 	if ds := manifest.RootEntry.DirSummary; ds != nil {
 		for _, ent := range ds.FailedEntries {
-			policy := policyTree.DefinedPolicy()
+			policy := policyTree.EffectivePolicy()
```

Comment:
Using the `DefinedPolicy` function here is improper as we don't pass the defined policy into [`policy.BuildTree(nil, &defaultPolicy)`](https://github.com/vmware-tanzu/velero/blob/v1.12.0-rc.1/pkg/uploader/kopia/snapshot.go#L86) which will result in getting one nil policy so that the logic will go into L250.

If we use EffectivePolicy, [which means the final take effective policy under the action of defined policy and default policy.](https://github.com/kopia/kopia/blob/v0.13.0/snapshot/policy/policy_tree.go#L154-L174)